src/schema/users.ts

```diff
@@ -967,6 +1025,55 @@ export const resolvers: IResolvers<any, Context> = {
         )
         .andWhere('"userId" = :userId', { userId: ctx.userId })
         .execute(),
+    subscribePersonalizedDigest: async (
```

Comment:
Who calls this?
Just wondering if the hour/day should be defaulted and optional

Reply:
Frontend from settings, I'll update once I open that PR.

Reply:
All are optional and filled up with default values in db if not sent. f212961d1e4a2466bc3df937812741646a985c0d